internal/testrunner/runners/system/runner.go

```diff
@@ -207,10 +231,53 @@ func NewSystemRunner(options SystemRunnerOptions) *runner {
 		runTestsOnly:               options.RunTestsOnly,
 		runTearDown:                options.RunTearDown,
 	}
+	r.resourcesManager = resources.NewManager()
+	r.resourcesManager.RegisterProvider(resources.DefaultKibanaProviderName, &resources.KibanaProvider{Client: r.kibanaClient})
+
+	r.serviceStateFilePath = filepath.Join(testrunner.StateFolderPath(r.profile.ProfilePath), testrunner.ServiceStateFileName)
 	// TODO: check if logic in initRun could be moved to this constructor
 	return &r
 }
 
+// SetupRunner prepares global resources required by the test runner.
+func (r *runner) SetupRunner(ctx context.Context) error {
+	if r.runTearDown {
+		logger.Debug("Skip installing package")
+	} else {
```

Comment:
Nit.
```suggestion
		return nil
	}
```

Reply:
This could have the condition inverted and then the else block removed.

Reply:
Hi @efd6 , that change to remove the else block was done in https://github.com/elastic/elastic-package/pull/1892/commits/68dfcb24e5e525d10f18e1aeb29f7b64e77a45c0 keeping the same condition.